src/vos/vos_pool.c

```diff
@@ -1734,14 +1748,33 @@ pool_open_post(struct umem_pool **p_ph, struct vos_pool_df *pool_df, unsigned in
 		goto out;
 	}
 
+	DLCK_PRINT(dp, "Containers tree...\n");
+	dlck_print_indent_inc(dp);
 	/* Cache container table btree hdl */
-	rc = dbtree_open_inplace_ex(&pool_df->pd_cont_root, &pool->vp_uma,
-				    DAOS_HDL_INVAL, pool, &pool->vp_cont_th);
+	rc = dbtree_open_inplace_ex(&pool_df->pd_cont_root, &pool->vp_uma, DAOS_HDL_INVAL, pool, dp,
+				    &pool->vp_cont_th);
 	if (rc) {
+		dlck_print_indent_dec(dp);
+		DLCK_PRINT(dp, "Containers tree... ");
+		DLCK_PRINT_RC(dp, rc);
 		D_ERROR("Container Tree open failed\n");
 		goto out;
 	}
 
+	/** This check is conducted only for the DLCK's purpose. No need to do it otherwise. */
+	if (IS_DLCK(dp)) {
+		rc = dlck_dbtree_check(pool->vp_cont_th, dp);
+		if (rc != DER_SUCCESS) {
+			dlck_print_indent_dec(dp);
+			DLCK_PRINT(dp, "Containers tree... ");
+			DLCK_PRINT_RC(dp, rc);
+			goto out;
+		}
+	}
+	dlck_print_indent_dec(dp);
+	DLCK_PRINT(dp, "Containers tree... ");
+	DLCK_PRINT_OK(dp);
```

Comment:
It seems that we introduce many `DLCK_PRINT()` lines when open_pool, that works, but make the logic to be more some verbose. Is it possible to wrap multiple `DLCK_PRINT()` into single call?